chia/data_layer/download_data.py

```diff
@@ -81,9 +82,13 @@ async def insert_into_data_store_from_file(
             serialized_node = SerializedNode.from_bytes(serialize_nodes_bytes)
 
             node_type = NodeType.TERMINAL if serialized_node.is_terminal else NodeType.INTERNAL
-            await data_store.insert_node(node_type, serialized_node.value1, serialized_node.value2)
 
-    await data_store.insert_root_with_ancestor_table(tree_id=tree_id, node_hash=root_hash, status=Status.COMMITTED)
+            print(f"insert node. Serialized:{serialized_node}")
```

Comment:
printf is only for resolve the issue of test. when resolve is possible to remove